src/drivers/bmi055/bmi055.hpp

```diff
@@ -447,7 +447,7 @@ class BMI055_accel : public BMI055
 	*
 	* @return true if the sensor is not on the main MCU board
 	*/
-	bool            is_external() { return (_bus == EXTERNAL_BUS); }
+	bool            is_external() { return (get_device_bus() == EXTERNAL_BUS); }
```

Comment:
In thinking about this, what makes a bus external? A board config know the answer. Can we consolidate the the notion of external and internal into the the Device or do you think the driver instance needs to carry this notation so it can be set via the command line?

Reply:
Then pushing it into the Device base class sounds perfect.